app/assets/javascripts/discourse/app/components/modal/create-account.hbs

```diff
@@ -31,244 +31,244 @@
       </div>
 
       {{#if this.showCreateForm}}
-        <div class="login-form">
-          <form>
-            {{#if this.associateHtml}}
-              <div class="input-group create-account-associate-link">
-                <span>{{html-safe this.associateHtml}}</span>
-              </div>
-            {{/if}}
-            <div class="input-group create-account-email">
-              <Input
-                {{on "focusout" (action "checkEmailAvailability")}}
-                @type="email"
-                @value={{this.model.accountEmail}}
-                disabled={{this.emailDisabled}}
-                autofocus="autofocus"
-                aria-describedby="account-email-validation"
-                aria-invalid={{this.emailValidation.failed}}
-                name="email"
-                id="new-account-email"
-                class={{value-entered this.model.accountEmail}}
+        <form id="login-form">
```

Comment:
Ah no, probably went wrong in the refactor – shouldnt be there, thanks!

Reply:
So I haven't changed anything to the template logic, and it seems to be like it's missing something?

```
    {{#if (and this.showLoginButtons this.site.desktopView)}}
      {{#unless this.canLoginLocal}}
        <div class="login-left-side">
          <Modal::Login::WelcomeHeader @wavingHandURL={{this.wavingHandURL}} />
        </div>
      {{/unless}}
      <div class="login-right-side">
        <LoginButtons
          @externalLogin={{this.externalLoginAction}}
          @passkeyLogin={{this.passkeyLogin}}
        />
      </div>
    {{/if}}
```

Should this have another {{if}} to render only when it has alternate login methods allowed? 

instead of hiding with css:

<img width="1370" alt="image" src="https://github.com/discourse/discourse/assets/101828855/a15f9520-aac2-47a8-bfec-5df16644971a">

Feels more clean to me.

Reply:
I added in this
```
{{#if this.hasAtLeastOneLoginButton}}
        <div class="login-right-side">
          <LoginButtons
            @externalLogin={{this.externalLoginAction}}
            @passkeyLogin={{this.passkeyLogin}}
          />
        </div>
      {{/if}}
```

prop seems to be what i need